packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -559,14 +580,14 @@ public void attachToContainer(ViewGroup viewGroup,
                                   OnGutenbergDidSendButtonPressedActionListener onGutenbergDidSendButtonPressedActionListener,
                                   ShowSuggestionsUtil showSuggestionsUtil,
                                   OnMediaFilesCollectionBasedBlockEditorListener onMediaFilesCollectionBasedBlockEditorListener,
-                                  OnFocalPointPickerTooltipShownEventListener onFocalPointPickerTooltipListener,
```

Comment:
👋 @SiobhyB , not sure, is the removal of the `onFocalPointPickerTooltipListener` parameter intentional? It seems to introduce a compile error since `onFocalPointPickerTooltipListener` is then not found in line 601. 🤔

Reply:
@hypest, I _believe_ that came about by mistake following a previous merge conflict. I had fixed the issue in my local environment but hadn't committed, done that now! Thanks for the ping. :)